examples/hostpath-provisioner/go.mod

```diff
@@ -3,10 +3,11 @@ module sigs.k8s.io/kubernetes-sigs/sig-storage-lib-external-provisioner/examples
 go 1.13
 
 require (
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/klog v1.0.0
+	github.com/evanphx/json-patch v4.2.0+incompatible // indirect
+	k8s.io/api v0.19.0-rc.2
```

Comment:
can we use non rc 1.19?

Reply:
Go picked this automatically. I didn't investigate why and ignored it because I have seen cases where different tags refer to the same source code. But this isn't the case here, so yes, let's better use v0.19.1.

I've pushed an updated commit with that.

Reply:
not seeing the new commit?

Reply:
Ooops. Sorry, I pushed the wrong branch. Now it's here.